bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataClientWrapper.java

```diff
@@ -46,39 +60,112 @@ public BigtableDataClientWrapper(BigtableDataClient bigtableDataClient,
         );
   }
 
+  /** {@inheritDoc} */
   @Override
   public void mutateRow(RowMutation rowMutation) {
     MutateRowRequest mutateRowRequest = rowMutation.toProto(requestContext);
     delegate.mutateRow(mutateRowRequest);
   }
 
+  /** {@inheritDoc} */
   @Override
-  public ApiFuture<Void> mutateRowAsync(RowMutation rowMutation) {
-    throw new UnsupportedOperationException("Not implemented yet");
+  public ListenableFuture<Void> mutateRowAsync(RowMutation rowMutation) {
+    ListenableFuture<MutateRowResponse> response =
+        delegate.mutateRowAsync(rowMutation.toProto(requestContext));
+    return  Futures.transform(response, new Function<MutateRowResponse, Void>() {
+      @NullableDecl
```

Comment:
What's NullableDecl?

Reply:
According to javadocs
>Identical to {@code @Nullable}, but can only be written at declaration locations.

It was an autogenerated annotation for `ApiFunction`, would change this to `@Nullable` for consistency across the repository.